tests/test_rate_limiter.py

```diff
@@ -1,53 +1,122 @@
 import responses
 import pytest
 import re
+from time import sleep
 from freezegun import freeze_time
 from app import app, settings
 from apistar.test import TestClient
 from idunn.blocks.wikipedia import WikipediaLimiter
-from time import sleep
 
-from .test_full import fake_all_blocks
+from .test_api import louvre_museum
+
 
 @pytest.fixture(scope="function")
-def limiter_test(redis):
+def limiter_test_normal(redis):
     """
-    We define here settings specific to tests
-    for the rate limiter in order to avoid
-    any waste of time
+    We define here settings specific to tests.
+    We define low max calls limits to avoid
+    too large number of requests made
     """
     settings._settings['WIKI_API_RL_PERIOD'] = 5
     settings._settings['WIKI_API_RL_MAX_CALLS'] = 3
     settings._settings['WIKI_API_REDIS_URL'] = redis
     WikipediaLimiter._limiter = None
-    wiki_limiter = WikipediaLimiter.init_limiter()
     yield
-    settings._settings['WIKI_API_RL_PERIOD'] = 100
-    settings._settings['WIKI_API_RL_MAX_CALLS'] = 1
+    settings._settings['WIKI_API_RL_PERIOD'] = 1
```

Comment:
don't you want to set it back to the old param values ?
else this will create a test side effect